pkg/operator/validations.go

```diff
@@ -19,6 +19,17 @@ import (
 	"github.com/prometheus/common/model"
 )
 
+// ValidRelabelActions for relabelling
+var ValidRelabelActions = map[string]bool{
```

Comment:
(nit) a more idiomatic way is to use a map of strings to empty structs.

```suggestion
var ValidRelabelActions = map[string]struct{}{
```